src/big_query.mli

```diff
@@ -163,6 +163,19 @@ module Jobs : sig
     ; cache_hit : bool
     }
 
+  val query_response_row_of_yojson :
+    ([> `Assoc of
+        (string * [> `List of [> `Assoc of (string * 'a) list ] list ]) list
+     | `Bool of bool
+     | `Float of float
+     | `List of [> `Assoc of (string * 'a) list ] list
+     | `Null
+     | `String of string ]
+     as
+     'a) ->
+    (query_response_row, string) result
```

Comment:
Use the named type here (it'll be either `Yojson.Safe.t` or `Yojson.Basic.t`, not 100% sure which).

You might want to use the "anonymous" variant type if we wanted to avoid an explicit dependency on `yojson`, but since we already depend on it there's no need.

```suggestion
  val query_response_row_of_yojson :
    Yojson.Safe.t ->
    (query_response_row, string) result
```